fraim/core/workflows/format_pr_comment.py

```diff
@@ -33,7 +33,7 @@
 {%- endif %}
 {%- endfor %}
 
-**Confidence**: {{ risk.properties.confidence }}%
+**Confidence**: {{ risk.properties.confidence * 10 }}%
```

Comment:
lol I just raised a PR to take this out entirely, I'll close mine, but maybe we should just remove it altogether? I don't see a point in exposing it to the end user.